### PR TITLE
Update Indonesian Translation Bundle

### PR DESCRIPTION
**Pull request by Kenny-Ofc**
Each issue that i've fixed :
- Rewrite the existing translation from lasted PR ( from WilloIzCitron ) https://github.com/Anuken/Mindustry/pull/9035/commits/2ff854016c6b4693980c2c914349dc816f7c29a7
- Each string line are messed up, i'd reorganized it by read it again from a top. ( it is also get helped by lasted changes from WilloIzCitron )
- The names for faction, turrets, cores and units is supposed to remain the same.  Example [ "Baluarti"] are actually spanish
wanna havin fun with momok are probably best idea, but yea i presume the name was the part from April Fools ]
- Change "armor" into it's international word "pelindung"
- Reposition some words that based on ( Butuh, Perlu, Tenaga, Daya, Barang, Benda, Sumber Daya, Bijih, Anda, Kamu, Target, Sasaran, Pipa, Saluran, Proyektor, Pangkalan, Pendaratan Musuh, Buat, Cipta, Distribusi, Penembak, Berlebih, Berkurang, Jumlah, Dapat, Dipindahkan, Memulihkan, Terlindungi, Peroleh, Darah, Proses Muatan, \ > atau, Suplai, Simpul, Memindahkan, Petak, Pengarah, and more !)
- Specific block with "optional" word = turn into ( Opsional ) at the end description.
- "Fabricator and Refabricator" word are now called "Pabrikator dan Pabrikator Ulang"
- Arkycite Liquid are STILL being the unexisted thing. But i want to make sure to keep the name until there is more information of said Liquid in Real World.
- Rewrite the Description of sectors and blocks with the V7 Version. Note that i've seeing old translation from V5
- Subjects such as The Tutorial Stuff, some said probably get changes. But anyway, submit this for next Mindustry Patch. (i would appreciate that)

### CODE REVIEW

**Review by DaGamerFiles — CHANGES_REQUESTED** [review]

**Comment by Kenny-Ofc** [comment]
notice typo ? it would get fixed until i manage to clears out my eyes with many Translation reference from Mindustry Mod

**Comment by Kenny-Ofc** [comment]
Revision is needed, this is for make ID Translation looks Better. Based on Playstore Review, i had motivation to keep working on this translation.

File: core/assets/bundles/bundle_id_ID.properties [hunk 1]
```diff
@@ -35,171 +35,175 @@ load.mod = Mod
...
+invalid = Tidak Valid
 pickcolor = Pilih Warna
 preparingconfig = Menyiapkan Konfigurasi
 preparingcontent = Menyiapkan Konten
 uploadingcontent = Mengunggah Konten
 uploadingpreviewfile = Mengunggah File Tinjauan
 committingchanges = Membuat Perubahan
 done = Selesai
-feature.unsupported = Perangkat Anda tidak mendukung fitur ini.
+feature.unsupported = Perangkat anda tidak mendukung fitur ini.
```
Comment (DaGamerFiles) [inline core/assets/bundles/bundle_id_ID.properties:130]: "Anda" should always have its first letter capitalized according to Indonesian orthography general guide (_Pedoman Umum Ejaan Bahasa Indonesia_/PUEBI)

**Review by DaGamerFiles — CHANGES_REQUESTED** [review]

File: core/assets/bundles/bundle_id_ID.properties (hunk 1 above)
Comment (WilloIzCitron) [reply core/assets/bundles/bundle_id_ID.properties]: Hmm, why not use Enhanced Indonesian Spelling (EYD) instead of it?

**Comment by Kenny-Ofc** [comment]
And.. that's it, well i have another work on different time line so, I'm probably open up this PR.

**Comment by WilloIzCitron** [comment]
I think it would be ready to open the pull request...

**Pull request merged**